src/tools/dev/scripts/build_visit

```diff
@@ -15,7 +15,7 @@
 #   so that this script can be improved in the future.
 #
 
-# To create your own modular file these 7 functions need to be implemented..
+# To create your own modular file these 13 functions need to be implemented..
 #filename: bv_<module>.sh
 #bv_<module>_initialize : Initialize any variables you may want to export
 #bv_<module>_enable : Enables the module
```

Comment:
Is this info documented in our docs? I'll hunt for it

Reply:
I cannot find it. @brugger1 should this be added to the documentation? i.e. should there be a new section about how to add a module? Or is this information fine where it is, at the top of `build_visit`?